frontend/containers/DocumentEditToolbar/DocumentEditToolbar.jsx

```diff
@@ -74,9 +75,14 @@ class DocumentEditToolbar extends Component {
   constructor(props) {
     super(props)
 
+    this.keyboard = new Keyboard()
     this.onSave = null
   }
 
+  componentDidMount() {
+    this.keyboard.on('cmd+s').do(this.saveDocument.bind(this))
```

Comment:
I would do:

```
this.keyboard.on('cmd+s').do(this.handleSave.bind(this, 'save'))
```

Because:

- `saveDocument` is supposed to be called by `handleSave`, not the other way around
- Removes the need for the "Apply default" block you've added to `saveDocument`
- Makes it more explicit which save mode is attached to Cmd+S